tests/models/test_base.py

```diff
@@ -9,121 +10,199 @@
 
 _TORCH_DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
 
-BATCH_SIZE = 4
+BATCH_SIZE = 2
 HEIGHTS = [128, 256, 384]  # standard numbers, not going to bigger images due to memory
 WIDTHS = [120, 246, 380]  # similar but not square
-BACKBONES = ["resnet18", "resnet34", "resnet50", "resnet101", "resnet152"]
+RESNET_BACKBONES = ["resnet18", "resnet34", "resnet50", "resnet101", "resnet152"]
+EFFICIENTNET_BACKBONES = ["efficientnet_b0", "efficientnet_b1", "efficientnet_b2"]
+VIT_BACKBONES = ["vit_b_sam"]  # "vit_h_sam" very large (2.6GB), takes too long to download/load
```

Comment:
`vit_h_sam` is expected to have 2.6GB and long downloading. that's how it is with big models and depends on internet connection.
If we were to commit to transformers (sam or not) we might want to move to float16 represntation instead of float32. this will eliminate memory by half and will allow for bigger nets and batchsizes.